src/entities.rs

```diff
@@ -2,14 +2,14 @@
 // this file is autogenerated by
 // curl https://html.spec.whatwg.org/entities.json | python generate_entities.py
 
-pub struct CharRef {
+pub(crate) struct CharRef {
```

Comment:
Just to double check, you didn't manually fix this file, right?

Reply:
Nope. I updated the .py file that generates this. (And rustfmt reformatted this file afterwards.)

Reply:
perfect, thanks